src/components/validations/validation-icon.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ValidationIcon from './validation-icon.component';
+import { InputPresentationContext } from '../../__experimental__/components/input';
+
+describe('ValidationIcon', () => {
+  it('renders with an icon for the given type', () => {
+    const wrapper = mount(<ValidationIcon type='error' />).find('div');
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('shows the tooltip if context has focus', () => {
+    const wrapper = mount(
+      <InputPresentationContext.Provider value={ { hasFocus: true } }>
+        <ValidationIcon type='error' />
+      </InputPresentationContext.Provider>
+    ).find('Icon');
+    expect(wrapper.props().tooltipVisible).toEqual(true);
```

Comment:
could you create a const `icon` and then test it's prop? At first sight it looks like you're checking the prop of the mounted component.